src/inc/unreachable.h

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ---------------------------------------------------------------------------
+// unreachable.h
+// ---------------------------------------------------------------------------
+
+
+#ifndef __UNREACHABLE_H__
+#define __UNREACHABLE_H__
+
+#if defined(_MSC_VER) || defined(_PREFIX_)
+#if defined(_TARGET_AMD64_)
+// Empty methods that consist of UNREACHABLE() result in a zero-sized declspec(noreturn) method
+// which causes the pdb file to make the next method declspec(noreturn) as well, thus breaking BBT
+// Remove when we get a VC compiler that fixes VSW 449170
+# define __UNREACHABLE() DebugBreak(); __assume(0);
```

Comment:
@janvorli Would you mind if I change it to `{ DebugBreak(); __assume(0); }`?

Reply:
No problem, please feel free to do that.